### PR TITLE
Validation: Fix duplicate descriptions  

### PR DESCRIPTION
**Pull request by annmariehitchcock**
This PR addresses missing and duplicate descriptions.

### CODE REVIEW

File: api-reference/beta/resources/security-datasource.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+title: ediscovery-dataSource resource type
+description: Describes the dataSource resource type with its methods, properties, relationships, and a JSON representation.
 author: "SeunginLyu"
 ms.localizationpriority: medium
 ms.prod: "ediscovery"
 doc_type: resourcePageType
 ---
 
-# dataSource resource type
+# ediscovery-dataSource resource type
```
Comment (MelanieHom) [inline api-reference/beta/resources/security-datasource.md:10]:
```suggestion
# ediscovery - dataSource resource type
```

I would add spaces between to emphasize that the object is ediscovery and its resource type is dataSource.

File: api-reference/beta/resources/security-datasource.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
 ---
-title: "dataSource resource type"
-description: "The dataSource entity is an abstract base class used to identify sources of content for eDiscovery."
+title: ediscovery-dataSource resource type
```
Comment (MelanieHom) [inline api-reference/beta/resources/security-datasource.md:2]: See [comment](https://github.com/microsoftgraph/microsoft-graph-docs/pull/17331/files#r900477981).

**Comment by annmariehitchcock** [comment]
**Global effort to fix build validation errors**
 @Lauragra 
The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

File: api-reference/beta/resources/security-datasource.md [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
+title: ediscovery - dataSource resource type
+description: Describes the dataSource resource type with its methods, properties, relationships, and a JSON representation.
 author: "SeunginLyu"
 ms.localizationpriority: medium
 ms.prod: "ediscovery"
 doc_type: resourcePageType
 ---
 
-# dataSource resource type
+# ediscovery - dataSource resource type
```
Comment (Lauragra) [inline api-reference/beta/resources/security-datasource.md:10]: Please do not update the H1 titles. We don't differentiate topic titles by API version or subnamespace. This API will be deprecated shortly in favor of the APIs in the security subnamespace.

File: api-reference/beta/resources/provisioningerrorinfo.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ---
-title: "provisioningErrorInfo resource type"
-description: "Describes the status of the provisioning event and the associated errors."
+title: provisioningErrorInfo resource type
+description: Describes the provisioningErrorInfo resource type with its properties and a JSON representation.
```
Comment (Lauragra) [inline api-reference/beta/resources/provisioningerrorinfo.md:3]: Global: Are these updated descriptions helpful? Generally, in the description field we provide the summary description of the resource, although in a few cases there were errors with the automation. In search results, won't it be more useful for users to get the description of the resource, as opposed to a description of the structure of the topic, which is the same for all resource types? This is reference content so every resource topic has the same H2 heading structure.

Comment (MelanieHom) [reply api-reference/beta/resources/provisioningerrorinfo.md:3]: Hi, @Lauragra. Thank you for your input. Just for some background information, we generally follow the Microsoft Contributor Guide for our edits. You can find the guidelines for meta descriptions [here](https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-meta-description?branch=main).

However, if it'll be more useful to the user to keep the original (with the quotes for search purposes), we can revert those changes. Let me and @annmariehitchcock know how you would like to proceed. Thank you for your time and concern.

**Comment by annmariehitchcock** [comment]
Descriptions, Headers and Title issues have all been resolved. Please re-review. Thank you!

**Comment by annmariehitchcock** [comment]
#feedback-addressed

**Review by Lauragra — APPROVED** [review]

**Pull request closed without merging**